src/js/modules/core/icons/PropagateLoader.js

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Yeah...let's not have two CSS in JS libraries. Can you rewrite this in react-jss?